www/views/hardware.html

```diff
@@ -1204,11 +1204,20 @@ <h1 data-i18n="Settings">Settings</h1>
 						<br />
 						<span data-i18n="Using CRC message validation prevents incorrect data from entering the Domoticz database.">Using CRC message validation prevents incorrect data from entering the Domoticz database.</span>
 						<br />
-						<strong><font class="warning-text"><span data-i18n="Should normally not be disabled!"></span>Should normally not be disabled!</font></strong>
+						<strong><font class="warning-text"><span data-i18n="Should normally not be disabled!"></span></font></strong>
 					</td>
 				</tr>
 			</table>
 		</div>
+    <div id="divratelimitp1">
+			<br>
+			<table class="display" id="hardwareparamsratelimitp1" border="0" cellpadding="0" cellspacing="20">
+				<tr>
+					<td align="right" style="width:110px"><label for="ratelimitp1"><span data-i18n="Rate Limit"></span>Rate Limit (sec):</label></td>
```

Comment:
Not sure what you have done here
Think this should be:

<td align="right" style="width:110px"><label for="ratelimitp1"><span data-i18n="Rate Limit">Rate Limit</span>:</label></td>

then the value

on the right of the value

Underneath/right of the value a little more help

<span data-i18n="(Seconds) 0 = Disabled"></span>

Translation is already available for '(Seconds) 0 = Disabled'